src/Workspaces/MSBuild/Test/NetCoreTests.cs

```diff
@@ -94,6 +96,27 @@ public async Task TestOpenProject_NetCoreApp()
         Assert.Empty(diagnostics);
     }
 
+    [ConditionalFact(typeof(DotNetSdkMSBuildInstalled))]
+    [Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+    [Trait(Traits.Feature, Traits.Features.NetCore)]
+    public async Task TestOpenInMemoryProject_NetCoreApp()
+    {
+        CreateFiles(GetNetCoreAppFiles());
+
+        var projectFilePath = GetSolutionFileName("Project.csproj");
+        var content = File.ReadAllText(projectFilePath);
```

Comment:
It seems like after you read this in, you should delete the file. Otherwise there's no guarantee you aren't still depending on the file's existence...